packages/emotion-primitives/src/css.js

```diff
@@ -0,0 +1,101 @@
+// @flow
+import transform from 'css-to-react-native'
+import { StyleSheet } from 'react-primitives'
+import { interleave } from './utils'
+
+export function css(...args: any) {
+  let vals
+
+  if (args[0] == null || args[0].raw === undefined) {
+    vals = args
+  } else {
+    vals = interleave(args)
+  }
+
+  let styles = []
+  let buffer = ''
+  let lastType
+
+  let handleInterpolation = (interpolation: *, i: number, arr: Array<*>) => {
+    let type = typeof interpolation
+
+    if (type === 'function') {
+      if (this === undefined && process.env.NODE_ENV !== 'production') {
+        console.error(
+          'Interpolating functions in css calls is not allowed.\n' +
+            'If you want to have a css call based on props, create a function that returns a css call like this\n' +
+            'let dynamicStyle = (props) => css`color: ${props.color}`\n' +
+            'It can be called directly with props or interpolated in a styled call like this\n' +
+            'let SomeComponent = styled.View`${dynamicStyle}`'
+        )
+      } else {
+        handleInterpolation(interpolation(this.mergedProps), i, arr)
+      }
+      return
+    }
+    let isFalsy = interpolation == null || type === 'boolean'
```

Comment:
this doesn't catch `false`
```js
false == null // false
```

Reply:
It has `type === 'boolean'` so it does catch false but I should probably rename it since it doesn't actually mean falsy it's meant to mean something that we don't care about.

Reply:
oh, the name threw me off 😉